packages/contracts-bedrock/test/universal/BenchmarkTest.t.sol

```diff
@@ -28,105 +29,17 @@ contract SetPrevBaseFee_Test is CommonTest {
     }
 }
 
-contract GasBenchMark_L1CrossDomainMessenger is CommonTest {
```

Comment:
We don't have any critical gas dependencies on the L1CrossDomainMessenger so remove this makes sense